packages/framework-info/tests/__snapshots__/main.test.ts.snap

```diff
@@ -186,16 +186,16 @@ exports[`getFrameworks > Should return the version of each framework when multip
       "port": 8080,
     },
     "env": {},
-    "id": "vuepress",
+    "id": "vue",
     "logo": {
-      "dark": "https://framework-info.netlify.app/logos/vuepress/default.svg",
-      "default": "https://framework-info.netlify.app/logos/vuepress/default.svg",
-      "light": "https://framework-info.netlify.app/logos/vuepress/default.svg",
+      "dark": "https://framework-info.netlify.app/logos/vue/default.svg",
+      "default": "https://framework-info.netlify.app/logos/vue/default.svg",
+      "light": "https://framework-info.netlify.app/logos/vue/default.svg",
     },
-    "name": "VuePress",
+    "name": "Vue.js",
     "package": {
-      "name": "vuepress",
-      "version": "4.5.6",
+      "name": "@vue/cli-service",
+      "version": "1.2.3",
```

Comment:
This is an example of what was wrong, vuepress, should be returned first, before vue.